### PR TITLE
Sync with upcoming zigpy release

### PR DESCRIPTION
**Pull request by puddly**
## Proposed change

Zigpy's new `attribute_updated` and ZCL schema changes break unit tests. The Sinope leak sensor also had two attributes called `zone_status`.

## Checklist

- [x] The changes are tested and work correctly
- [x] `pre-commit` checks pass / the code has been formatted using Black
- [ ] Tests have been added to verify that the new code works

### CODE REVIEW

File: zhaquirks/sinope/sensor.py [hunk 1]
```diff
@@ -45,7 +45,7 @@ class SinopeManufacturerCluster(CustomCluster):
 class SinopeTechnologiesIasZoneCluster(CustomCluster, IasZone):
     """SinopeTechnologiesIasZoneCluster custom cluster."""
 
-    class ZoneStatus(t.enum8):
+    class LeakStatus(t.enum8):
         """zone_status values."""
```
Comment (javicalle) [inline zhaquirks/sinope/sensor.py:49]: """leak_status values"""

**Review by javicalle — CHANGES_REQUESTED** [review]
Fix comment

File: zhaquirks/sinope/sensor.py [hunk 2]
```diff
@@ -45,16 +45,16 @@ class SinopeManufacturerCluster(CustomCluster):
...
+        """leak_status values."""
 
         Dry = 0x00
         Leak = 0x01
 
     attributes = IasZone.attributes.copy()
     attributes.update(
         {
-            0x0030: ("zone_status", ZoneStatus, True),
+            0x0030: ("leak_status", LeakStatus, True),
```
Comment (TheJulianJES) [inline zhaquirks/sinope/sensor.py:57]: Haven't checked ZHA yet, but did this quirk rely on HA creating a binary sensor for an attribute called `zone_status`? (And possibly the higher attribute id overwriting the actual `zone_status`?)

So, is another "binary sensor entity" necessary in ZHA for this `leak_status` now?

Comment (puddly) [reply zhaquirks/sinope/sensor.py:57]: You're right, it does rely on an IASZone `zone_status` attribute existing. I'll add a new matcher should to ZHA in the corresponding PR.

**Pull request merged**